quisby/pricing/cloud_pricing.py

```diff
@@ -177,8 +216,14 @@ def get_cloud_cpu_count(instance_name, region, cloud_type):
 
 
 if __name__ == "__main__":
-    # region = "us-east"
     # print(get_azure_pricing("Standard_D32s_v3",region))
-    # print(get_aws_cpucount("i3en.24xlarge",region))
     # print(get_gcp_prices("n2-standard-16",region)
-    pass
+    region = "us-east-1"  # Replace with your desired AWS region
```

Comment:
I'd assume we can pass these in on the command line rather than editing the python file? </stupidquestion>